udata/frontend/csv.py

```diff
@@ -79,7 +89,11 @@ def to_row(self, obj):
         '''Convert an object into a flat csv row'''
         row = []
         for name, getter in self.get_fields():
-            row.append(safestr(getter(obj)))
+            try:
+                row.append(safestr(getter(obj)))
+            except Exception, e:  # Catch all errors intentionally.
+                log.error('Error exporting CSV for {name}: {error}'.format(
```

Comment:
I think this will create shorter rows so invalid CSV, we need to append something to the row